draft-ietf-quic-transport.md

```diff
@@ -5492,15 +5504,14 @@ ACK Delay:
 
 ACK Range Count:
 
-: A variable-length integer specifying the number of Gap and ACK Range fields in
+: A variable-length integer specifying the number of ACK Range fields in
```

Comment:
This whole section was a bit awkward.  I tried to operate on the principle that the Ack Range is the compound structure, and that Gap and the Ack Range Length are the member fields within it.  The resulting text isn't very pretty, though...

Reply:
This is a good improvement -- thanks!